packages/apps/halo-app/src/pages/JoinIdentityPage.tsx

```diff
@@ -6,14 +6,15 @@ import React, { useCallback } from 'react';
 import { useNavigate, useSearchParams } from 'react-router-dom';
 
 import type { Invitation } from '@dxos/client';
+import { JoinPanel } from '@dxos/react-appkit';
```

Comment:
So the existing "os" patterns have moved into appkit? I know in general apps shouldn't be using those patterns but halo would be the exception to that because it would be providing them. Presumably the successor to these will live in `@dxos/react-ui` though, right? We wouldn't want these to live in appkit because then it would give all of our apps access to them.

Reply:
Right, there's still nontrivial tech debt in `JoinPanel` and some other OS patterns that landed in `appkit`, so I've moved them there for the time being.

Apps may still want something like `JoinPanel`, even if the OS might have its own version, so this is one pattern where having the two styles available could be helpful.